op_crates/web/02_event.js

```diff
@@ -916,7 +916,7 @@
           // If signal is not null and its aborted flag is set, then return.
           return;
         } else {
-          // If listener’s signal is not null, then add the following abort
```

Comment:
This character non-ascii?

Reply:
Really subtle but this is a *right single quotation mark* (0x2019).
A quick `s/’/'/g` and we should be good.